frontend/src/router.tsx

```diff
@@ -35,169 +44,186 @@ const AppRouter: React.FC = () => {
-        title={getTitle('Map View')}
-      />
-      <AppRoute
-        protected
-        path="/properties/list"
-        component={PropertyListView}
-        layout={AuthLayout}
-        claim={Claims.PROPERTY_VIEW}
-        title={getTitle('View Inventory')}
-      />
-      <AppRoute
-        protected
-        path="/dispose"
-        component={ProjectDisposeView}
-        layout={AuthLayout}
-        claim={Claims.PROJECT_ADD}
-        title={getTitle('Dispose Property')}
-      />
-      <AppRoute
-        protected
-        path="/projects/list"
-        component={ProjectListView}
-        layout={AuthLayout}
-        claim={Claims.PROJECT_VIEW}
-        title={getTitle('View Projects')}
-      />
-      <AppRoute
-        protected
-        path="/projects/approval/requests"
-        component={ProjectApprovalRequestListView}
-        layout={AuthLayout}
-        claim={Claims.DISPOSE_APPROVE}
-        title={getTitle('Surplus Property Program Projects - Approval Requests')}
-      />
-      <AppRoute
-        protected
-        path="/projects"
-        component={ProjectRouter}
-        layout={AuthLayout}
-        claim={Claims.PROJECT_ADD}
-        title={getTitle('Dispose Property')}
-      />
-      <AppRoute
-        protected
-        path="/project/submitted"
-        component={ProjectDisposalSubmitted}
-        layout={AuthLayout}
-        claim={Claims.PROJECT_VIEW}
-        title={getTitle('Dispose Property Submitted')}
-      />
-      <AppRoute
-        protected
-        path="/project/exemption/submitted"
-        component={ProjectDisposalExemptionSubmitted}
-        layout={AuthLayout}
-        claim={Claims.PROJECT_VIEW}
-        title={getTitle('Dispose Property Submitted')}
-      />
-      <AppRoute
-        protected
-        path="/admin/user/:id?"
-        component={EditUserPage}
-        layout={AuthLayout}
-        claim={Claims.ADMIN_USERS}
-        title={getTitle('Edit User')}
-      />
-      <AppRoute
-        protected
-        path="/admin/agencies"
-        component={ManageAgencies}
-        layout={AuthLayout}
-        claim={Claims.ADMIN_USERS}
-        title={getTitle('Agency Management')}
-      />
-      <AppRoute
-        protected
-        path="/admin/agency/:id"
-        component={EditAgencyPage}
-        layout={AuthLayout}
-        claim={Claims.ADMIN_USERS}
-        title={getTitle('Edit Agency')}
-      />
-      <AppRoute
-        protected
-        path="/admin/agency/new"
-        component={EditAgencyPage}
-        layout={AuthLayout}
-        claim={Claims.ADMIN_USERS}
-        title={getTitle('Edit Agency')}
-      />
-      <AppRoute
-        protected
-        path="/reports/spl"
-        component={SplReportContainer}
-        layout={AuthLayout}
-        claim={Claims.REPORTS_SPL}
-        title={getTitle('SPL Reports')}
-      />
-      <AppRoute title="*" path="*" component={() => <Redirect to="/page-not-found" />} />
-    </Switch>
+    <Suspense
```

Comment:
Suspense will display the default app layout with a spinner when lazy-loading components.